### PR TITLE
Fix cargo text alignment

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
The cargo text labels are spaced too far from the vehicle sprite. This was introduced sometime between v24.06 and v24.07; I'm assuming by #2513.

v24.06:
Screenshot-v24 06

v24.07:
Screenshot-v24 07

This PR:
Screenshot

### CODE REVIEW

File: src/OpenLoco/src/Ui/Windows/Vehicle.cpp [hunk 1]
```diff
@@ -1966,8 +1966,8 @@ namespace OpenLoco::Ui::Windows::Vehicle
                         {
                             cargoTextHeight += 5;
                         }
-                        drawCargoText(drawingCtx, width, cargoTextHeight, strFormat, body->primaryCargo.qty, body->primaryCargo.type, body->primaryCargo.townFrom);
-                        drawCargoText(drawingCtx, width, cargoTextHeight, strFormat, front->secondaryCargo.qty, front->secondaryCargo.type, front->secondaryCargo.townFrom);
+                        drawCargoText(drawingCtx, width - 24, cargoTextHeight, strFormat, body->primaryCargo.qty, body->primaryCargo.type, body->primaryCargo.townFrom);
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/Ui/Windows/Vehicle.cpp:1969]: Is it really 24 fixed and not some other variable value.

Comment (AaronVanGeffen) [reply src/OpenLoco/src/Ui/Windows/Vehicle.cpp:1969]: Yeah it's literally offsetting the sprite by -24 a few lines up. Maybe could use a constant? But what to name it?

Comment (AaronVanGeffen) [reply src/OpenLoco/src/Ui/Windows/Vehicle.cpp:1969]: Doh, it's actually much simpler. The text should just be at x=24 at all times.

**Review by duncanspumpkin — APPROVED** [review]
do we want to changelog this?

**Pull request merged**